Xamarin.Forms.Platform.WPF/FormsPanel.cs

```diff
@@ -82,6 +82,13 @@ protected override System.Windows.Size MeasureOverride(System.Windows.Size avail
 			if (double.IsInfinity(elementDesiredWidth) || double.IsPositiveInfinity(elementDesiredHeight))
 			{
 				Size request = Element.Measure(elementDesiredWidth, elementDesiredHeight, MeasureFlags.IncludeMargins).Request;
+				
+				if (request.Width == -1)
```

Comment:
You have another pr with a different fix right?

Reply:
yes, i've struggled a little with this github thing and ended up pulling it in the wrong branch, sorry.